apps/proxy-auth/src/app/otp/send-otp/send-otp.component.ts

```diff
@@ -63,7 +64,7 @@ export class SendOtpComponent extends BaseComponent implements OnInit, OnDestroy
     public otpWidgetData;
     public showRegistration = new BehaviorSubject<boolean>(false);
     public referenceElement: HTMLElement = null;
-    public showLogin: boolean = false;
+    public showLogin = new BehaviorSubject<boolean>(false);
```

Comment:
Use subject from service instead of creating new
it will give error for default UI